Fix: PR checklist is missing item for operator to check upgrading instructions (#5161) (#5162)
DataBiosphere/azul: .github/pull_request_template.md

- [ ] Checked `reindex` label and `r` commit title tag
- [ ] Checked that demo expectations are clear <sub>or all connected issues are labeled `no demo`</sub>

> - [ ] Checklist items accompanying upgrading instructions are present <sub>or PR lacks `upgrade` label</sub>

Reviewer: ```suggestion
- [ ] PR has checklist items for upgrading instructions <sub>or PR is not labeled `upgrade`</sub>
```
